applications/views.py

```diff
@@ -178,3 +183,13 @@ def post(self, request, *args, **kwargs):
             c = self.get_context_data()
             c.update({'form': form})
             return render(request, self.template_name, c)
+
+
+@login_required
+def save_draft(request):
```

Comment:
Seeing this makes me think someone could attack this by saving other fields such as `STATUS`. I don't think it's possible to send those with the application when the user submits. Wanted to flag just in case.

Reply:
Fair point. Will narrow down to fields on model and on form. That should make it safe

Reply:
It should still not be able as draftapplication is only used to fill fields in the form between sessions. But it's not too bad to narrow down so that we don't forget about it later on.